op-acceptance-tests/tests/interop/message/interop_msg_test.go

```diff
@@ -0,0 +1,27 @@
+package msg
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/interop"
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+// TestInitExecMsg tests basic interop messaging
+func TestInitExecMsg(gt *testing.T) {
```

Comment:
Looks great! I added a todo for me to merge this with `TestInteropHappyTx` from https://github.com/ethereum-optimism/optimism/pull/15662 as they are almost equivalent in tests/checks, with this one being a lot more readable / succinct.